jenkins/docker/Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS') 
+    }
+    agent none
+    stages {
+        stage('parameters') {
+            steps {
+                script {
+                    properties([
+                            parameters([
+                                    string(
+                                            defaultValue: 'https://github.com/opensearch-project/opensearch-build',
+                                            name: 'DOCKER_BUILD_GIT_REPOSITORY',
+                                            description: 'The git repository name that contains dockerfiles and the docker build script',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: 'main',
+                                            name: 'DOCKER_BUILD_GIT_REPOSITORY_REFERENCE',
+                                            description: 'The git reference (branch/tag/commit_id) of above repository',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: 'bash docker/ci/build-image-multi-arch.sh -v <TAG_NAME> -f <DOCKERFILE PATH>',
```

Comment:
Rather than have this as parameters to the job can we have the job pickup the value from elsewhere so it is all in source control?

Reply:
We can make it a choice param or retrieve from a manifest file, that can be an improvement step.

Reply:
Could you update #801 to include what improvements we are yet to make or create a new issue?

Reply:
Added.